.isort.cfg

```diff
@@ -0,0 +1,25 @@
+# https://github.com/timothycrosley/isort/wiki/isort-Settings
+[settings]
+default_section = THIRDPARTY
+# force_to_top=file1.py,file2.py
+# forced_separate = django.contrib,django.utils
+include_trailing_comma = true
+indent = 4
+known_first_party = proxy
+# known_future_library = future,pies
+# known_standard_library = std,std2
+known_testing_party = pytest,unittest
+# length_sort = 1
+# Should be: 80 - 1
+#line_length = 79
+line_length = 127
+lines_after_imports = 2
+# https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html
+# NOTE: Another mode could be "5" for grouping multiple "import from" under
+# NOTE: a single instruction.
+multi_line_output = 9
```

Comment:
@abhinavsingh this one corresponds best to the style you already use in the repo but I usually prefer `5`: https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html

Reply:
It's not a limit, it's a style (or "mode") of formatting, follow the lint and you'll see. aiohttp actually uses `3` for some reason but I didn't set that up and try not to change the formatting settings in that project.

Reply:
"isort your imports, so you don't have to."

I see what you mean now.  It's part of a fixture config :).

Happy to see this integration, indeed following a standard is the way to go, I was sorting them based upon a random in-my-brain-only logic 🗡️

Reply:
Yeah, also one of the most common import orders is to have stdlib at the top, followed by third-party imports and finally project-local. But with isort you can define extra sections if some things have additional semantical meaning within the project or explicitly declare that some things belong to certain sections.